build/optaplanner-build-parent/pom.xml

```diff
@@ -31,6 +31,10 @@
     <version.com.h2database>1.3.173</version.com.h2database>
     <version.com.thoughtworks.xstream>1.4.14</version.com.thoughtworks.xstream>
     <version.io.quarkus>1.10.0.Final</version.io.quarkus>
+    <!-- keep these aligned with gizmo -->
+    <version.io.quarkus.gizmo>1.0.4.Final</version.io.quarkus.gizmo>
+    <version.org.jboss.jandex>2.1.3.Final</version.org.jboss.jandex>
```

Comment:
(important) Out of sync with our quarkus bom version quarkus-bom 1.10.0.Final has
```

        <jandex.version>2.2.1.Final</jandex.version>
       ...
        <gizmo.version>1.0.6.Final</gizmo.version>
```